test-utils/transfer-wrapper/src/wrapper.rs

```diff
@@ -395,7 +395,8 @@ impl TransferWrapper {
             .collect();
 
         let mut outputs = vec![];
-        let output_value = input - fee.gas_limit - crossover_value;
+        let output_value =
+            input - fee.gas_limit * fee.gas_price - crossover_value;
```

Comment:
This change should not be in a commit prefixed with `transfer-contract: `

Reply:
Will fix